src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -53,7 +55,11 @@ public partial class MauiDrawingView
 	/// <summary>
 	/// Used to draw any shape on the canvas
 	/// </summary>
-	public Action<ICanvas, RectF>? DrawAction { get; set; }
+	public Action<ICanvas, RectF>? DrawAction
```

Comment:
FYI, if someone uses this property, the caller would need to store the delegate in a member field, or the GC will collect it.

Some examples on this here: 

* https://github.com/dotnet/maui/wiki/Memory-Leaks#when-we-cant-use-weakeventmanager

I would write a test to ensure this callback still works after the GC runs.